engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -342,7 +342,7 @@ namespace dmGameSystem
         {
             HashMaterial(&state, resource->m_Materials[i].m_Material);
 
-            dmHashUpdateBuffer32(&state, resource->m_Materials[i].m_Textures, resource->m_Materials[i].m_TexturesCount);
+            dmHashUpdateBuffer32(&state, resource->m_Materials[i].m_Textures, sizeof(dmGameSystem::MaterialTextureInfo)*resource->m_Materials[i].m_TexturesCount);
```

Comment:
Let's hash the entire array, not just the first few bytes (We read 1 byte per texture which won't work)